docs/dev-guide/android-sdk.md

```diff
@@ -370,68 +371,90 @@ Please see `JitsiMeetActivity`, which registers for all the events and can serve
 #### Supported events
 
 ##### CONFERENCE_JOINED
-Broadcasted when a conference was joined.
-The `data` HashMap contains a `url` key with the conference URL.
+
+Broadcasted when a conference was joined. `data` contains the following information:
+
+- `url`: the conference URL
 
 ##### CONFERENCE_TERMINATED
-Broadcasted when the active conference ends, be it because of user choice or
-because of a failure.
-The `data` HashMap contains an `error` key with the error and a `url` key
-with the conference URL. If the conference finished gracefully no `error`
-key will be present.
+
+Broadcasted when the active conference ends, be it because of user choice or because of a failure. `data` contains the
+following information:
+
+- `url`: the conference URL
+- `error`: missing if the conference finished gracefully, otherwise contains the error message
 
 ##### CONFERENCE_WILL_JOIN
-Broadcasted before a conference is joined.
-The `data` HashMap contains a `url` key with the conference URL.
+
+Broadcasted before a conference is joined. `data` contains the following information:
+
+- `url`: the conference URL
 
 ##### AUDIO_MUTED_CHANGED
-Broadcasted when audioMuted state changed.
-The `data` HashMap contains a `muted` key with state of the audioMuted for the localParticipant.
 
-##### VIDEO_MUTED_CHANGED
-Broadcasted when videoMuted state changed.
-The `data` HashMap contains a `muted` key with state of the videoMuted for the localParticipant.
+Broadcasted when the local participant's audio is muted or unmuted. `data` contains the following information:
+
+- `muted`: a boolean indicating whether the audio is muted or not.
 
 ##### PARTICIPANT_JOINED
-Broadcasted when a participant has joined the conference.
-The `data` HashMap contains information of the participant that has joined.
-Depending of whether the participant is the local one or not, some of them are 
-present/missing.
-    isLocal
-    email
-    name
-    participantId
+
+Broadcasted when a participant has joined the conference. `data` contains the following information:
+
+- `email`: the email of the participant.
```

Comment:
In my example app, the `email` key never existed. Should this be removed?

Reply:
It may not be set if the remote participant didn't set one.